### DISCUSSION TITLE
Error while adding boolean, float, integer or smallint field to my entity

### DISCUSSION THREAD

**Discussion by Anis-J** [symfony/symfony#53791] [Q&A]
I start with symfony, 
I create an entity called product using make:entity
I added name and description fields respectively string and text
I execute make:migration and doctrine:migrations:migrate and every thing is ok I got my table product with tow fields.

Now I try to add a field boolean called isDispo and I got this error :(

php bin/console make:entity
Class name of the entity to create or update (e.g. FiercePizza):
 > Product
 Your entity already exists! So let's add some new fields!
 New property name (press <return> to stop adding fields):
 > isDispo
 Field type (enter ? to see all types) [boolean]:
 > 
 Can this field be null in the database (nullable) (yes/no) [no]:
 > 
**In DoctrineHelper.php line 262:
Undefined constant Doctrine\DBAL\Types\Types::ARRAY**  
make:entity [-a|--api-resource] [-b|--broadcast] [--regenerate] [--overwrite] [--] [<name>]

I tried to add boolean, float, integer or smallint field and always the same error !!
Can you help me with that please

**Comment by MatTheCat** [comment]
This was fixed by https://github.com/symfony/maker-bundle/pull/1438; youʼll have to wait for the maker bundleʼs next release.

**Comment by Anis-J** [comment]
@MatTheCat Thank you for your answer.

**Comment by Anis-J** [comment]
Error fixed by [git-released] v1.54.0
